sdk/tools/spec2def/spec2def.c

```diff
@@ -495,6 +517,14 @@ PrintName(FILE *fileDest, EXPORT *pexp, PSTRING pstr, int fDeco)
     const char *pcName = pstr->buf;
     int nNameLength = pstr->len;
     const char* pcDot, *pcAt;
+    char namebuffer[16];
+
+    if ((nNameLength == 1) && (pcName[0] == '@'))
+    {
+        sprintf(namebuffer, "ordinal%d", pexp->nOrdinal);
```

Comment:
With `nOrdinal` being an `int`, the buffer for this should be 7+10+1=18 bytes

Reply:
Is `int` even the right type for that? I see `ULONG` being used for ordinals in ntdll and freeldr's PE loader while other sources like https://web.archive.org/web/20130601160643/http://win32assembly.programminghorizon.com/pe-tut7.html claim that it's a "16-bit number".
Let's first decide about the right type here before we base further buffer size decisions on this type.

Reply:
ordinals are USHORT. With MAX_USHORT being invalid ordinal. I'll change the type.

Reply:
Reverted back to int and using 19 here. The implicit conversions just cause too much trouble.